.github/workflows/test_gdal_tags.yaml

```diff
@@ -0,0 +1,43 @@
+name: Test against recent GDAL tags
+
+on:
+  schedule:
+    - cron: '0 3,15 * * *'
+  pull_request:  # also build on PRs touching this file
+    paths:
+      - ".github/workflows/test_gdal_tags.yaml"
+      - ".github/workflows/test_gdal_build.yaml"
+      - "ci/gdal-compile.sh"
+      - "ci/get-new-tags.py"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  get-tags:
+    name: Find recent tags and generate test job matrix
+    runs-on: ubuntu-latest
+    outputs:
+      tags: ${{ steps.fetch-tags.outputs.tags }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
3.11 has a critical fix in `datetime.datetime.fromisoformat()` needed to parse `atom:updated` in general cases.